coreos-postinst

```diff
@@ -340,10 +353,37 @@ if [[ -z "${LDSO}" ]]; then
 fi
 LIBS="${LDSO%/*}"
 
-call_cgpt() {
-    "${LDSO}" --library-path "${LIBS}" "${CGPT}" "$@"
+
+# Use torcx binary from next-OS image to ensure compatibility
+TORCX_BIN=""
+for bindir in bin/old_bins bin sbin lib/coreos; do
+    if [[ -x "${INSTALL_MNT}/${bindir}/torcx" ]]; then
+        TORCX_BIN="${INSTALL_MNT}/${bindir}/torcx"
+        break
+    fi
+done
+if [[ -z "${TORCX_BIN}" ]]; then
+    echo "Failed to locate torcx binary in ${INSTALL_MNT}" 2>&1 | tee_journal
+    exit 1
+fi
+
+call_torcx() {
+    "${TORCX_BIN}" "$@" 2>&1 | tee_journal
 }
 
+# Check if torcx requires any additional addon available on a remote.
+${TORCX_CHECK_REMOTE_ONLY:="true"}
```

Comment:
When setting a default value like this, use something like the form `: "${var:=value}"`.  Otherwise if that variable is set in the environment, bash will try to execute its value.